src/gobimport/enricher/bag.py

```diff
@@ -29,6 +55,102 @@ def __init__(self, app_name, catalogue_name, entity_name):
             "standplaatsen": self.extract_nevenadressen,
         })
 
+    def _get_filename(self, name):
+        """Gets the full filename given a the name of a file
+
+        :param name:
+        :return:
+        """
+        dir = tempfile.gettempdir()
+        # Create the path if the path not yet exists
+        temp_filename = os.path.join(dir, name)
+        os.makedirs(os.path.dirname(temp_filename), exist_ok=True)
+        return temp_filename
+
+    def _download_amsterdam_sleutel_file(self, catalogue_name, entity_name):
+        """Download the Amsterdamse sleutel file from the objectstore
+
+        :param catalogue_name:
+        :param entity_name:
+        :return:
+        """
+        connection, user = connect_to_objectstore(get_objectstore_config('Basisinformatie'))
+        container_name = os.getenv("CONTAINER_BASE", "acceptatie")
```

Comment:
Maybe move the retrieving and setting of default value of env vars to config.py

Reply:
CONTAINER_BASE is now imported from config.py